docs/source/release/v6.2.0/mantidworkbench.rst

```diff
@@ -8,52 +8,57 @@ Mantid Workbench Changes
 New and Improved
 ----------------
 
-- Superplot is a new decorator widget for the plot window. It facilitates over-plotting and manipulation of overplotted data. See :ref:`Superplot documentation <WorkbenchSuperplot>` for more information.
+- **Superplot is a new decorator widget for the plot window. It facilitates over-plotting and manipulation of overplotted data. See** :ref:`Superplot documentation <WorkbenchSuperplot>` **for more information.**
 
 .. figure:: ../../images/superplot_1.png
     :width: 500px
     :align: center
 
-- Workflow diagrams in help pages are now ``.svg`` rather than ``.png``
+- **New widget and workbench plugin:** :ref:`WorkbenchWorkspaceCalculator` **, allows to perform binary operations and scaling by a floating number on workspaces;
```

Comment:
Missing a word here `New widget and workbench plugin: :ref:WorkbenchWorkspaceCalculator, allows (you/the user/users)`